bridge/ApiRequestHandler.cpp

```diff
@@ -435,21 +444,23 @@ httpd::Response ApiRequestHandler::onRequest(const httpd::Request& request)
                 }
                 else
                 {
-                    throw httpd::RequestErrorException(httpd::StatusCode::METHOD_NOT_ALLOWED);
+                    throw httpd::RequestErrorException(httpd::StatusCode::METHOD_NOT_ALLOWED,
+                            utils::format("HTTP method '%s' not allowed on this endpoint", request._methodString.c_str()));
```

Comment:
Yes, I can use an empty message and handle this differently. On the other hand this would be the only error that has a different response format, other option could be simplify the  message to "Method not allowed" just to keep the response format consistent with other requests.

Let me know what do you prefer, remove completely the response body or keep the consistency and perhapses simplify the message as the error is quite obvious.

Reply:
You have a point to keep it consistent.